GNUmakefile

```diff
@@ -31,6 +32,7 @@ bin: tools
 
 # dev creates binaries for testing locally - these are put into ./bin and $GOPATH
 dev:
+	vendorfmt
```

Comment:
We could ditch the `format` target and add `@go fmt $(GOFILES)` here too, though I guess most folks have their editors doing this anyway.